go.mod

```diff
@@ -94,7 +94,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 	k8s.io/api v0.26.11
 	k8s.io/apiextensions-apiserver v0.26.10
-	k8s.io/apimachinery v0.26.11
+	k8s.io/apimachinery v0.29.3
```

Comment:
Is that necessary in this PR? IIRC, updating this affect the kubernetes compatibility. It's probably better to update all k8s.io dependency to be in v0.29 in a dedicated PR if necessary.

Reply:
Will revert this.